test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewComponentTests.cs

```diff
@@ -140,6 +140,63 @@ public void ViewComponent_View_WithModelParameter_SetsResultViewWithDefaultViewN
         }
 
         [Fact]
+        public void ViewComponent_View_WithNullModelParameter_SetsResultViewWithDefaultViewNameAndNullModel()
+        {
+            // Arrange
+            var viewComponent = new TestViewComponent();
+            viewComponent.ViewData.Model = new object();
+
+            // Act
+            var actualResult = viewComponent.View(model: null as object);
```

Comment:
Why test a variation that is identical from MVC's perspective? The difference matters only to the compiler and few, if any customers will choose this anyhow.